services/config/pmm-managed.yaml

```diff
@@ -32,3 +32,6 @@ services:
       retry_backoff_env: "PERCONA_TEST_TELEMETRY_RETRY_BACKOFF"
       retry_count: 20
       send_timeout: 5s
+  pprof:
+    profile_duration: 30s
+    trace_duration: 10s
```

Comment:
Let's not introduce new fields to this config. There is a high chance that we will remove it soon. These values may be defined just as constants in service that uses them.

Reply:
Moved to constants but I have a doubts about it: every change of duration value require rebuild whole application.

Reply:
This config embedded into the binary during compilation. So you need to rebuild application anyway.